frontend/src/Editor/Inspector/Elements/Code.jsx

```diff
@@ -14,7 +16,24 @@ export const Code = ({
   fxActive,
   component,
 }) => {
-  const initialValue = definition ? definition.value : '';
+  const getDefinitionForNewProps = (param) => {
+    if (['showAddNewRowButton', 'allowSelection'].includes(param)) {
+      if (param === 'allowSelection') {
```

Comment:
Need to check `allowSelection` is undefined here. Else `allowSelection` value will always be depend on `bulkSelection` and `highlightSelectedRow`